packages/server/src/RoomManager.ts

```diff
@@ -0,0 +1,31 @@
+import { Server as IOServer, Socket } from 'socket.io';
+
+const MAX_ROOM_SIZE = 2;
+
+export class RoomManager {
```

Comment:
I'm totally newbie in nodejs but do we really need classes here? Is there any other way?

Reply:
Yeah don't know... Will do some research. For now, a class is not really needed much anyway, so made two functions.